findmyparadise/src/main/java/com/tp/wrc/findmyparadise/models/Listing.java

```diff
@@ -3,7 +3,6 @@
 import javax.persistence.*;
 
 @Entity
-@Table(name = "Listing")
```

Comment:
@Table is necessary, just not (name = "Listing") since the table will inherit the name of the Java class unless you wanted to make it something different than "Listing". Sorry for being nit-picky.